### ISSUE TITLE
Fix Demo Bugs

### ISSUE THREAD

**Issue by rockboynton** [Senior-Design-0x07/the-hobby-hub#38]
# Overview

We need to fix the issues that arose when trying to demo for the Winter presentation

# Goals

Bug fix
* Package wasn't working
* Program manager wasn't working
* frontend and backend weren't connected

# Helpful Info

# Verification Criteria

Can demo how originally planned 
1. Connect BB to WiFi 
2. See pin manager and program manager showing test programs being run in frontend
3. Log in to VS code dev environment as dev user
4. Create new program using package
5. Add to program manager
6. Go back to frontend, refresh, and see updated pin manager and program manager

# Task Advisor

@Ivendrakeo

**Comment by rockboynton** [comment]
@anthony-bartman Are steps 1 and 2 working yet?

**Comment by anthony-bartman** [comment]
@rockboynton  No.... Andy is working on reading text files for his wifi scripts, Declan, don't know where he is at, but I will be seeing exactly what he has on Monday. My pin manager page can connect to backend, but I have to add a few functional things which we will chat about tomorrow.
